engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
```

Comment:
In the end I've decided not to use the `.cads-grid-col-md-4.layout__sidebar` wrapper because there is a case in the public-website where the component is used within the 8-col layout (within the mobile toggle) https://github.com/citizensadvice/public-website/blob/develop/app/views/advice/_advice_collection.html.haml#L10, and I also saw there can be other content in that container ([preview placeholder and expandable sidebar](https://github.com/citizensadvice/public-website/blob/develop/app/views/advice/_advice_collection.html.haml#L39)) Please do let me know however, if I should use the wrapper classes here

Reply:
That makes sense, those wrapper classes really site outside this component anyway